### ISSUE TITLE
Examine SimplifyJS for contour maps

### ISSUE THREAD

**Issue by johntyree** [enthought/tabs#63]
I suspect we could fix some of the performance issues around the salt contour map visualization if we filtered out points before rendering them.

http://mourner.github.io/simplify-js/

**Comment by kjordahl** [comment]
Good idea, but the problem is topology. If two regions share a line as a border, simplifying them separately will not in general keep the shared border correct. There are tools (e.g. [topojson](https://github.com/mbostock/topojson)) to help with this.

Simplification could also be done on the python side ([contours_to_geoJSON()](https://github.com/enthought/tabs/blob/4cd2da664d4c778dd960f52a0f4ead6f6fb6b74b/tabs/thredds_frame_source.py#L121)) if we know what resolution we want.
